Sources/Sentry/SentryScope.m

```diff
@@ -97,7 +97,7 @@ - (instancetype)initWithScope:(SentryScope *)scope
         [_fingerprintArray addObjectsFromArray:[scope fingerprints]];
         [_attachmentArray addObjectsFromArray:[scope attachments]];
 
-        self.propagationContext = [[SentryPropagationContext alloc] init];
+        self.propagationContext = scope.propagationContext;
```

Comment:
I will open an extra PR for this change before merging this PR, as it's a small bug fix that deserves it's own PR.

Reply:
Extra PR is here: https://github.com/getsentry/sentry-cocoa/pull/4518. I will wait to merge this PR before merging this one.